woocommerce/woocommerce-ios
# Blaze: update status check to use `me/sites` endpoint - Storage layer changes

Part of #9866 


## Why


Previously, we implemented the Blaze status check using the `blaze/status` endpoint. As Jorge discovered in pe5sF9-1yI-p2#comment-2327, we can use the existing `me/sites` or `sites/$siteID` endpoint to check it for better performance as it saves the app some API requests.

This PR just contains the Storage layer changes since the model version changes are already 897 diffs.

## How

In a new model version 91, added `isAdmin` and `canBlaze` attributes to `Site` entity.

## Testing instructions


- From `trunk`, build the app and log in if needed
- From the PR branch, build the app and make sure the app runs as before

---
